src/common/checksums.cpp

```diff
@@ -355,11 +356,11 @@ void ValidateChecksumHeader::slotChecksumCalculated(const QByteArray &checksumTy
     const QByteArray &checksum)
 {
     if (checksumType != _expectedChecksumType) {
-        emit validationFailed(tr("The checksum header contained an unknown checksum type '%1'").arg(QString::fromLatin1(_expectedChecksumType)));
+        emit validationFailed(tr("The checksum header contained an unknown checksum type '%1'").arg(QString::fromLatin1(_expectedChecksumType)), QByteArray(), QByteArray(), QString());
```

Comment:
Same comment about missing `_filePath` and `{}`

Reply:
@mgallien what do we need a `_filePath ` for? It should be passed only in case we pass other parameters, as, I control this scenario on a slot's end to not process cases that I don't want (e.g. mismatch in checksum type).

Reply:
same stuff about logging inside slots connected to `validationFailed`